test/parallel/test-cluster-worker-isdead.js

```diff
@@ -7,11 +7,11 @@ if (cluster.isMaster) {
   const worker = cluster.fork();
   assert.ok(
     !worker.isDead(),
-    'isDead() should return false right after the worker has been created.');
+    `isDead() returned ${worker.isDead()}. isDead() should return false right after the worker has been created.`);
```

Comment:
The line length exceeds 80 characters. To make sure the function is only called a single time it would also be better to use a extra variable as in `const workerState = !worker.isDead()` and check for that.

Reply:
@sharpstef ... woo! first PR! This is super close but as @BridgeAR indicates there are some style nits that *can* be addressed by the person landing the PR but definitely feel free to address. Specifically what @BridgeAR is asking for would be a change to make it look like:

```js
const workerDead = worker.isDead;
assert.ok(!workerDead,
          `isDead() returned ${workerDead}. isDead() should return `  +
          'false right after the worker has been created.');
```

Reply:
On it